cmd/osbuild-composer/testdata/test.toml

```diff
@@ -5,3 +5,6 @@ ca = "/etc/osbuild-composer/ca-crt.pem"
 [worker]
 allowed_domains = [ "osbuild.org" ]
 ca = "/etc/osbuild-composer/ca-crt.pem"
+
+[weldr_api]
+image_type_denylist = [ "qcow2", "vmdk" ]
```

Comment:
Assuming there isn't a better way to do this (let the distro code do the filtering) this should be a mapping of distro name (or glob?) to image types. You may want to only disable `qcow2` on certain distros not on all of them.

Reply:
I see what you mean and it sounds interesting. My motivation was to have a mechanism to block a specific image types (soon to be defined RHEL EC2 images with new name) from being exposed via Weldr API regardless of the distro.

Unless you feel strongly about your proposal, I would prefer to keep the logic simple for now and don't allow specifying distro name or globing.